site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,57 +16,157 @@ updated: 2021-08-04
+[`chrome.scripting.executeScript`](/docs/extensions/reference/scripting/#method-executeScript).
+
+### Chrome 96: Expanded promise support {: #m96-promise-support }
+
+A number of general and Chrome OS-specific extensions APIs have gained promise support in this
+release. Expand the following sections for more details.
+
+{% Details %}
+{% DetailsSummary %}
+Extensions APIs
+{% endDetailsSummary %}
+
+A number of APIs now have promise support in Manifest V3.
+
+- [`chrome.browsingData`](/docs/extensions/reference/browsingData)
+- [`chrome.commands`](/docs/extensions/reference/commands)
+- [`chrome.contentSettings`](/docs/extensions/reference/contentSettings/)
+- [`chrome.debugger`](/docs/extensions/reference/debugger)
+- [`chrome.downloads`](/docs/extensions/reference/downloads)
+- [`chrome.enterprise.hardwarePlatform`](/docs/extensions/reference/enterprise_hardwarePlatform)
+- [`chrome.fontSettings`](/docs/extensions/reference/fontSettings)
+- [`chrome.history`](/docs/extensions/reference/history)
+- [`chrome.instanceID`](/docs/extensions/reference/instanceID)
+- [`chrome.permissions`](/docs/extensions/reference/permissions)
+- [`chrome.processes`](/docs/extensions/reference/processes)
+- [`chrome.search`](/docs/extensions/reference/search)
+- [`chrome.sessions`](/docs/extensions/reference/sessions)
+- [`chrome.signedInDevices`](/docs/extensions/reference/signedInDevices)
+- [`chrome.topSites`](/docs/extensions/reference/topSites)
+
+Also, APIs that use the [`ChromeSetting`](/docs/extensions/reference/types/#type-ChromeSetting)
+prototype now also support promises. The following APIs are affected by this change.
+
+- [`chrome.privacy`](/docs/extensions/reference/privacy/)
+- [`chrome.accessibilityFeatures`](/docs/extensions/reference/accessibilityFeatures/)
+- [`chrome.proxy`](/docs/extensions/reference/proxy/)
+
+{% endDetails %}
+
+{% Details %}
+{% DetailsSummary %}
+Chrome OS APIs
+{% endDetailsSummary %}
+
+- [`chrome.certificateProvider`](/docs/extensions/reference/certificateProvider)
+- [`chrome.documentScan`](/docs/extensions/reference/documentScan)
+- [`chrome.enterprise.deviceAttributes`](/docs/extensions/reference/enterprise_deviceAttributes)
+- [`chrome.enterprise.networkingAttributes`](/docs/extensions/reference/enterprise_networkingAttributes)
+- [`chrome.fileBrowserHandler`](/docs/extensions/reference/fileBrowserHandler)
+- [`chrome.fileSystemProvider`](/docs/extensions/reference/fileSystemProvider)
+- [`chrome.loginState`](/docs/extensions/reference/loginState)
+- [`chrome.printingMetrics`](/docs/extensions/reference/printingMetrics)
+- [`chrome.wallpaper`](/docs/extensions/reference/wallpaper)
+
+{% endDetails %}
+
+### Chrome 96: `declarativeNetRequestWithHostAccess` permission
+
+The new `declarativeNetRequestWithHostAccess` permission allows existing Manifest V2 extensions that
+leverage `webRequest`, `webRequestBlocking`, and site-specific host permission to migrate to the
```

Comment:
```suggestion
use `webRequest`, `webRequestBlocking`, and site-specific host permission to migrate to the
```

Reply:
The tech community likes to use 'leverage' as a synonym for 'use'. Unfortunately non-English speakers (and folks new to tech) won't find this definition in a dictionary and translations may or may not handle it correctly.

Reply:
Ack.